src/shared/lib/OncoKbUtils.ts

```diff
@@ -19,8 +20,9 @@ const ONCOGENIC_CLASS_NAMES:{[oncogenic:string]: string} = {
 // (used for sorting purposes)
 const ONCOGENIC_SCORE:{[oncogenic:string]: number} = {
```

Comment:
To be more specific, I am getting this TS error:
`error TS7015: Element implicitly has an 'any' type because index expression is not of type 'number'`